d2/src/main/java/com/linkedin/d2/balancer/properties/ClusterStoreProperties.java

```diff
@@ -113,16 +114,52 @@ public ClusterStoreProperties(String clusterName,
     super(clusterName, prioritizedSchemes, properties, bannedUris, partitionProperties, sslSessionValidationStrings, darkClusters, delegated);
     _canaryConfigs = canaryConfigs;
     _canaryDistributionStrategy = distributionStrategy;
+    _clusterFailoutProperties = null;
   }
 
-  public ClusterStoreProperties(ClusterProperties stableConfigs, ClusterProperties canaryConfigs, CanaryDistributionStrategy distributionStrategy)
+  public ClusterStoreProperties(String clusterName,
```

Comment:
Style nit, I believe the coding style for Pegasus prefers arguments on the same line with breaks only to stay under the line limit.

Reply:
The style guide is to minimize the line breaks, so have as many arguments on each line as possible, and break when it goes over the limit as John indicated.

Reply:
All of the other preexisting constructors in this file are using one line per argument; Should I go ahead and change all of them?

Reply:
If that's how the rest of the file is I think it's fine to leave them as-is